app_spec/zomes/blog/code/src/lib.rs

```diff
@@ -28,43 +32,43 @@ define_zome! {
         main (Public) {
             check_sum: {
                 inputs: |num1: u32, num2: u32|,
-                outputs: |post: JsonString|,
+                outputs: |post: ZomeApiResult<JsonString>|,
                 handler: blog::handle_check_sum
             }
 
-            hash_post: {
+            post_address: {
                 inputs: |content: String|,
-                outputs: |result: JsonString|,
-                handler: blog::handle_hash_post
+                outputs: |result: ZomeApiResult<Address>|,
```

Comment:
While we're changing the macro behavior, can we consider removing the requirement of naming inputs and outputs? Maybe as a separate PR? The names shouldn't matter, especially for the outputs, it's just noise. (right?)

Reply:
@maackle yes this bugs me too

inputs might be tricky because the named parameters map to a struct inside the macro, and it is pretty standard rust fare to have `name: Type`

outputs is always a single value so the name seems unneccessary

however, i think this is out of scope for this branch, we should put it somewhere in the SOA tree

Reply:
Started a discussion about it here: https://github.com/holochain/holochain-rust/issues/725